js/src/Dapps/dapps.js

```diff
@@ -76,72 +75,36 @@ class Dapps extends Component {
       );
     }
 
-    return (
-      <Page
-        title={
-          <FormattedMessage
-            id='dapps.label'
-            defaultMessage='Decentralized Applications'
-          />
-        }
-      >
-        { this.renderList(this.store.visibleLocal) }
-        { this.renderList(this.store.visibleViews) }
-        { this.renderList(this.store.visibleBuiltin) }
-        { this.renderList(this.store.visibleNetwork, externalOverlay) }
-      </Page>
-    );
-  }
+    const applications = [].concat(this.store.visibleLocal, this.store.visibleViews, this.store.visibleBuiltin, this.store.visibleNetwork);
 
-  renderList (items, overlay) {
     return (
-      <SectionList
-        items={ items }
-        noStretch
-        overlay={ overlay }
-        renderItem={ this.renderApp }
-      />
-    );
-  }
-
-  renderApp = (app) => {
-    if (app.onlyPersonal && this.props.availability !== 'personal') {
-      return null;
-    }
-
-    return (
-      <DappCard
-        app={ app }
-        key={ app.id }
-        showLink
-      />
+      <Page>
+        <div className={ styles.dapps }>
+          {
+            applications.map((app, index) => (
+              <DappCard
+                app={ app }
+                availability={ availability }
+                className={ styles.dapp }
+                key={ `${index}_${app.id}` }
```

Comment:
Wouldn't app.id be enough?

Reply:
Yes, overkill (as I typed it that thought came to mind)

Reply:
Actually no. Just "fixed", the "reverted" as I remembered why I took this longer route.

When I have the app as both a local app (linked from dapps) as well as an app of the network, the ids are the same. (It was not an issue previously since they were in separate lists)

Reply:
OK makes sense.